docs/en/reference/markdown-basics.md

```diff
@@ -124,6 +124,26 @@ Currently, the following languages are supported by the engine (the names in bra
 
 More languages can be implemented on your request. If you need a specific language, please [refer to the available ones](https://codemirror.net/mode/) and open up an issue on GitHub, so that we know which one we should add!
 
+## Zettlr Markdown additions
+
+In addition to GitHub flavored markdown extensions (marked with "(extension)" in the [spec](https://github.github.com/gfm/)), Zettlr provides the following:
+
+ - Support for `<iframe src="https://example.com></iframe>` elements
```

Comment:
I would add here that this is the only way to open up security issues because if you insert a malicious page, it might be possible for an attacker to "hijack" Zettlr (so something like "use with caution" would be fitting here)

Reply:
I hadn't thought through the security implications and I was a little surprised when I did, so I've put a very strong warning to try and communicate the severity.

Reply:
That's good! But I would precede the Warning with some reassurance that common pages are totally harmless, something like: "**Warning**: Make sure to only embed from pages you trust, such as YouTube, Vimeo, or Twitter. Pages in iFrames can get unrestricted … [etc]"

This way people will immediately know that YouTube etc. is safe (which will be the main use case either way) but still know what might happen if they get wild :)